### PR TITLE
#PBI 1234 added testing data

### PR DESCRIPTION
**Pull request by mohanckumar**
sad

### CODE REVIEW

File: EF6CodeFirstDemo/SchoolContext.cs [hunk 1]
```diff
@@ -7,6 +7,7 @@ public class SchoolContext : DbContext
         public SchoolContext() : base("SchoolDB-EF6CodeFirst")
         {
             Database.SetInitializer<SchoolContext>(new SchoolDBInitializer());
+
```
Comment (Rakshithkumar078) [inline EF6CodeFirstDemo/SchoolContext.cs:10]: Please remove the spaces make it allign.

**Review by Rakshithkumar078 — CHANGES_REQUESTED** [review]

File: EF6CodeFirstDemo/SchoolContext.cs (hunk 1 above)
Comment (mohanckumar) [reply EF6CodeFirstDemo/SchoolContext.cs:10]: sure ill make changes

Comment (Rakshithkumar078) [inline EF6CodeFirstDemo/SchoolContext.cs:10]: started review

**Review by Rakshithkumar078 — APPROVED** [review]

**Comment by Rakshithkumar078** [comment]
working

**Pull request merged**